kbase/narrative
# Dataup 224 adding checkboxes

# Description of PR purpose/changes

This is an in progress PR on the master ticket of DATAUP-224 -> adding multi-select checkboxes to the staging area. Once feedback on this PR is wrapped up my plan is to merge into the top level feature branch, and create a new PR for the rest of the work. Hopefully this way it's a bit more manageable to digest for reviewers. 

The bulk of the work in this PR is:

* refactoring CSS so that we have a more sane and structured BEM model 
* refactoring old datatables JS so that it's using the current recommend structure
* adding checkboxes to the data table
* making it so that checkboxes are disabled until a user selects a type
* moving sort control for the columns to the left of the column header (instead of right justified)
* removed upload/import button

What has NOT been done:

* any testing
* functionality for the select all checkbox
* there may still be some alignment issues and polishing to do 
* implement new import selected button and hook up functionality 
 
# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-224

# Testing Instructions
* Load a narrative and navigate to the data panel, import some files if you haven't already. 
* You should see disabled checkboxes until you select a type from the import as dropdown.

# Dev Checklist:
